Example/AblyChatExample/ContentView.swift

```diff
@@ -2,6 +2,34 @@ import Ably
 import AblyChat
 import SwiftUI
 
+// Set mode to `.live` if you wish to connect to actual instances of the Chat client in either Prod or Sandbox environments. Setting the mode to `.mock` will use the `MockChatClient`, and therefore simulate all features of the Chat app.
+private let mode = Environment.mock
+
+private enum Environment: Equatable {
+    case mock
+    /// - Parameters:
+    ///   - key: Your Ably API key.
+    ///   - clientId: A string that identifies this client.
+    case live(key: String, clientId: String)
+
+    func createChatClient() -> ChatClient {
+        switch self {
+        case .mock:
+            return MockChatClient(
+                realtime: MockRealtime.create(),
```

Comment:
I know it's not from this PR, but `MockRealtime.create()` looks excessive in comparing to just `MockRealtime()`

Reply:
Did you see the comment on the `create()` method?